### DISCUSSION TITLE
Feedback on GitHub Actions API Stability

### DISCUSSION THREAD

**Discussion by JeffreyPalmer** [community/community#24578] [Apps, API and Webhooks]
<p>I have some anecdotal feedback on the stability of the GitHub Actions API, in case you have any thoughts or suggestions for API consumers.</p>
<p>Requesting the following URL results in a Bad Gateway error for over 50% of requests I send to it, but this behavior seems to be limited to this project (storybook).</p>
<pre><code>https://api.github.com/repositories/54173593/actions/runs?per_page=30
</code></pre>
<p>This project has a very large build history, in case that’s relevant. I initially attempted to query 50 workflow runs per page, but that basically never worked and kept causing my http client to disable the github connection. 30 results per page is slightly better, but even so the failure rate is so high that I thought I’d raise the issue. For what it’s worth, I don’t see this behavior with any other projects.</p>
<p>Hope this is helpful!</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/jeffreypalmer">@jeffreypalmer</a>, thanks for being here. The best way to get feedback like this to GitHub is via the Feedback form found here: <a href="https://support.github.com/contact/feedback" rel="nofollow noopener">https://support.github.com/contact/feedback</a></p>
<p>Thanks for sharing your thoughts! </p>
